src/bobtail/subblock.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2020 The Bitcoin Unlimited developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "subblock.h"
+
+void CSubBlock::SetNull()
+{
+    CBlockHeader::SetNull();
+    vtx.clear();
+}
+
+bool CSubBlock::IsNull()
+{
+    return (vtx.empty() && this->IsNull());
```

Comment:
I'm probably missing something, but if `this` refers to the calling instance of `CSubBlock`, then wouldn't calling `this->IsNull()` result in an infinite loop? Or will `this` always grab the parent class method when inside the child class method?

Reply:
fixed to be more specific